### ISSUE TITLE
HexTags wipes scoreboard every round even with a config that doesn't define "ScoreTag"

### ISSUE THREAD

**Issue by g1xn** [Hexer10/HexTags#55]
"Chat-Processor" 2.2.1
"hextags" 1.50

TL;DR, I run a Multi1v1 Arena server and I used a "NoPrime" scoretag to indicate non-prime players, then due to player demand, I removed it from the HexTags config & now HexTags wipes the tags "Arena 1-16" on almost all players every-round.

"HexTags"
{
    "b"
    {
        "NameColor" "{lightred}"
    }
    
    "NoPrime"
    {
        "ScoreTag" "[F2P]"
    }
}

Even after going back to a config that doesn't use "ScoreTag" at all the scoretags are wiped every round.

"HexTags"
{
    "b"
    {
        "NameColor" "{lightred}"
    }
}

I think this may be related to a cache issue? I can't wrap my head around why HexTags is wiping the scoreboard without a config that says to, I've tried reinstalling chat-processor and HexTags but to no avail...

**Comment by donnie011** [comment]
My server has the same problem, ScoreTag does not work.
